cmd/flux/bootstrap_gitlab.go

```diff
@@ -85,6 +86,7 @@ type gitlabFlags struct {
 	repository      string
 	interval        time.Duration
 	personal        bool
+	visibility      string
```

Comment:
Please create a typed flag that contains the validation, see https://github.com/fluxcd/flux2/blob/main/internal/flags/decryption_provider.go

Reply:
Done.